addons/crm/models/crm_team.py

```diff
@@ -78,3 +117,66 @@ def action_your_pipeline(self):
             ]
         action['context'] = action_context
         return action
+
+    def _compute_dashboard_button_name(self):
+        for team in self:
+            if team.team_type != 'pos' and team.use_opportunities:
+                team.dashboard_button_name = _("Pipeline")
+            else:
+                super(Team, team)._compute_dashboard_button_name()
+
+    def action_primary_channel_button(self):
+        if self.team_type != 'pos' and self.use_opportunities:
```

Comment:
Yeah, but then I'd need to make a bridge module between pos_sale and crm then ^^